src/app/containers/Navigation/index.jsx

```diff
@@ -1,48 +1,81 @@
 import React, { useContext } from 'react';
-import Navigation, {
-  NavigationUl,
-  NavigationLi,
-} from '@bbc/psammead-navigation';
+import { NavigationUl, NavigationLi } from '@bbc/psammead-navigation';
+import { DropdownUl, DropdownLi } from '@bbc/psammead-navigation/dropdown';
 import { ServiceContext } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
+import Canonical from './index.canonical';
+import Amp from './index.amp';
+
+const renderListItems = (Li, navigation, script, currentPage, service, dir) =>
+  navigation.map((item, index) => {
+    const { title, url } = item;
+    const active = index === 0;
+
+    return (
+      <Li
+        key={title}
+        url={url}
+        script={script}
+        active={active}
+        currentPageText={currentPage}
+        service={service}
+        dir={dir}
+      >
+        {title}
+      </Li>
+    );
+  });
 
 const NavigationContainer = () => {
+  const { platform } = useContext(RequestContext);
+  const isAmp = platform === 'amp';
```

Comment:
I think there was a PR that added `isAmp` to RequestContext so we can use it directly instead of checking the platform now

Reply:
Cool! This has been merged, we can use it here.

Reply:
I've updated this to use the value from context